api-object-tagging.go

```diff
@@ -155,7 +155,8 @@ func (c Client) RemoveObjectTaggingWithContext(ctx context.Context, bucketName,
 	}
 
 	if resp != nil {
-		if resp.StatusCode != http.StatusOK {
+		// S3 returns "204 No content" after Object tag deletion.
+		if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusNoContent {
```

Comment:
We should perhaps only look for 204 and catch offending non compliant servers? @BigUstad

Reply:
I will make this check.